style.scss

```diff
@@ -44,6 +44,15 @@
     font: icon !important;
 }
 
+[is="vl-button-link"] [is="vl-icon"] {
+    color: #ffaa33;
+    mix-blend-mode: difference;
+
+    &:hover {
+        color: #ffc44f;
+    }
```

Comment:
De kleur in Chrome is niet blauw omdat de achtergrond op de demo pagina transparant is. Dit is op zich geen probleem omdat de body in de huisstijl een witte achtergrond heeft, dus uiteindelijk komt de kleur goed.